sw/host/opentitanlib/src/bootstrap/rescue.rs

```diff
@@ -259,8 +259,11 @@ impl Rescue {
         for _ in 0..Self::MAX_CONSECUTIVE_ERRORS {
             eprint!("Resetting...");
             container.reset_pin.write(false)?; // Low active
-            std::thread::sleep(container.reset_delay);
+            uart.write(&[3])?; // Send a character to ensure that HyperDebug UART->USB
+                               // forwarding has "woken up", see b/298075416.
```

Comment:
Yes, this is a link to a Google-internal bug tracker.  I thought about whether such a reference was appropriate in a open source project, but decided that since this particular file implements a legacy protocol used only by Google's previous Titan-class chips, it will mostly be Googlers who are interested in how and why it works, and having a reference explaining why this workaround was added will be useful to them.